File: SPA/GHR-App/src/app/models/funcionarios/FuncionarioMeta.ts
import { Meta } from "../metas/Meta";
import { Funcionario } from "./Funcionario";


export class FuncionarioMeta {
  id: number;
  metaId: number;
  meta: Meta;
  funcionarioId: number;
  funcionario: Funcionario;
  metaCumprida: Boolean;
  inicioAcordado?: string;
  fimAcordado?: string;
  inicioRealizadb: string;
  fimRealizado: string;
  supervisor: string;
}

File: Server/src/GHR.Domain/DataBase/Funcionarios/FuncionarioMeta.cs
using System;
using GHR.Domain.DataBase.Departamentos;
using GHR.Domain.DataBase.Metas;

namespace GHR.Domain.DataBase.Funcionarios
{
    public class FuncionarioMeta
    {
        public int MetaId { get; set; }
        public Meta Metas { get; set; }
        public int FuncionarioId { get; set; }
        public Funcionario Funcionarios { get; set; }
        public Boolean MetaCumprida { get; set; }
        public string InicioAcordado { get; set; }
        public string FimAcordado { get; set; }
        public string InicioRealizado { get; set; }
        public string FimRealizado { get; set; }
        public string Supervisor { get; set; }
        public Departamento Departamentos { get; set; }
    }
}